<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тяговый расчет</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Ukraine"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Груз поступающий на участок</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQgr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>т/час</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Qр1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQr1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Qр2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQr2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Qл</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditQl">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>м</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLenght">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Длинна участка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>градусов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBeta">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Угон наклона участка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getSectCmd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Добавить точку</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCountSect">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPROCHNOST">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>242</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>СЧИТАТЬ</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>30</y>
     <width>91</width>
     <height>171</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="lstNatjag">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>80</y>
     <width>71</width>
     <height>121</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>S уд</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMOSHNOST">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>40</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Мощность</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstNatjag2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>71</width>
     <height>121</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>Шир.ленты</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditWBelt">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>см</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>кг/м</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>кг/м</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>кг/м</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
   <property name="text">
    <string>м/с</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSpeed">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Russian" country="Ukraine"/>
   </property>
  </widget>
  <widget class="QGroupBox" name="MenOrMenHard">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbHard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Груз</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbMenHard">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Груз/Пас</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="POSITION">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QRadioButton" name="rbUp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вверху</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbDown">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Внизу</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditQr1</tabstop>
  <tabstop>lineEditQr2</tabstop>
  <tabstop>lineEditQl</tabstop>
  <tabstop>lineEditQgr</tabstop>
  <tabstop>lineEditLenght</tabstop>
  <tabstop>lineEditBeta</tabstop>
  <tabstop>lineEditWBelt</tabstop>
  <tabstop>getSectCmd</tabstop>
  <tabstop>lstNatjag</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>lstNatjag2</tabstop>
  <tabstop>lineEditPROCHNOST</tabstop>
  <tabstop>lineEditMOSHNOST</tabstop>
  <tabstop>lineEditCountSect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
